requirements.txt

```diff
@@ -9,4 +9,6 @@ awscli
 flake8
 python-dotenv>=0.5.1
 wget==3.2
-unzip==0.1
```

Comment:
Do you use these libraries in your feature? If not, then keep dependencies unchanged, please

Reply:
it is strange, but alright, let's keep it then